### DISCUSSION TITLE
How to define fraction eliminated in urine to a certain number?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1610] [How-Tos]
Hi!
I am reading an article:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9504927/
While they optimized the metabolism and excretion process, it says:
"Elimination via heparinase in plasma and renal elimination via glomerular filtration were accounted for in the model by fixing the fraction eliminated in urine to 40%, as reported in the FDA label."
I am wondering how is this done in PK-SIM? i.e. Fixing the fraction eliminated in urine to a certain number.
Looking forward for your help!
Many thanks!

**Comment by tobiasK2001** [comment]
Hi, you can include a datapoint in your Parameter identification for  Kidney-Urine-fraction excreted and set the weight to a high value (e.g. 40) like this: 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/82131743-fd8f-4bbb-9897-fee45fb80126)
Hope this might help.

Best, Tobias

**Reply by Gre-ux** [reply]
Seems reasonable！
Thanks for your reply！

Best, Greta

**Reply by AyaSaleh90** [reply]
Hi @tobiasK2001 and @Gre-ux 

I have a follow-up question regarding a drug primarily metaboliSed but with a small portion (approximately 2%) excreted unchanged in urine. Our model includes different CYP enzymes responsible for metabolism and we have included a process with a "GFR fraction = 1" to account for minor renal elimination.

However, when simulating the fraction excreted, it reaches up to 18%, which is much higher than the reported values. Is there a specific parameter in PK-Sim/MoBI, like "fraction excreted," that we can estimate to limit this process or fix it to the reported values? Any insights or recommendations on how to better control this aspect of the model would be greatly appreciated.

Thank you in advance for your help!
Best, Aya

**Reply by tobiasK2001** [reply]
Dear Aya,

Sorry for answering slowly.  It looks like your compound is reabsorbed in the proximal kidney tubules as a GFR < 1 is need to account for the fact that you find less than 18% fraction excreted to urine. You can fit like in the example mentioned above the Kidney-fraction excreted  to urine to your measured value. The identified GFR fraction should be lower than 1. A more complicated way leading to similar result would be to implement a active reuptake transport for your compound in the Kidney.

Hope you might find it useful.

Best, Tobias

**Reply by songsong02** [reply]
Dear Tobias，
I have also encountered the aforementioned problem. I want to fix the proportion of eliminating drugs in urine at 66%, but I don't have the observation data of urine excretion. So, what would you suggest? Are there any other methods? 
I am extremely eager to receive your reply. Thank you sincerely.
Best，song

**Reply by tobiasK2001** [reply]
Hi,
in that case you can create the data you need in an excel sheet setting like this and import to PK-Sim
![image](https://github.com/user-attachments/assets/50570087-0272-4756-98f1-65872d06ff69)

Than fit your simulated fraction dose excreta to that made up data using GFR as Identification - parameter. Eventually you need to increase the weight.

Hope this might help

**Reply by songsong02** [reply]
Thank you so much for your reply. Your advice means a lot to me, and I’ll definitely give it a try. 

Best,song
